project_proposals/liblrs.md

```diff
@@ -0,0 +1,136 @@
+* MIT
+
+Javascript binding and demo app:
+* MIT
+* BSD-3-Clause license
+* ISC license
+
+## Are any trademarks associated with the project?
+
+No.
+
+## Does the project have a web site? Where is it? Are you ok with moving it to be hosted by the OpenRail Association?
+
+There is a demonstration of the app hosted at https://osrd-project.github.io/liblrs/
+
+We agree to move it to the OpenRail Association.
+
+## What are the communication channels the project uses (such as mailing lists, Slack, IRC, etc.)?
+
+Matrix: https://matrix.to/#/#liblrs:osrd.fr
+
+
+## What is the project's leadership team and decision-making process?
+
+Product manager : loic.hamelin@sncf.fr
+Lead developer : ext.tristram.grabener@sncf.fr
+Product owner : ext.celine.durupt@sncf.fr
+
+As the team is small, all decisions are made collaboratively. We meet each week to share progress and decide on next steps. We are open to include any interested person into the decision-making team.
+
+## How is it decided if and when a pull request is merged?
+
+The review of a maintainer is required.
+
+## How can someone become a committer or a maintainer to/of the project?
+
+Anybody can suggest a pull request. If it is accepted after the review, one becomes a commiter.
+
+Maintainers are coopted by the other maintainers. Candidates must be an active commiter.
+
+## How is development of the project planned and organized? Is this transparent to the public?
+
+Every issue, pull request and review is public. Technical debates are done in the public matrix channel.
+
+## What is the project's roadmap?
+
+* Publish the library on crates.io, npm and pypi
+* Documentation for the various edge case
+* Interactive demonstration based on opendata
+* Use liblrs for the project OSRD https://github.com/OpenRailAssociation/osrd
+
+## What other organizations in the world should be interested in this project?
+
+Appart for railway infrastructure manager, any company working with linear infrastructure should be interested:
+* electrical network
+* pipeline operator
+* highway operator
+* touristical bike route coordinator 
+
+## Why would this project be a good candidate for inclusion in the OpenRail Association?
+
+This library has the ambition to be a cornerstone of any tool concerning rail networks. As such, being included in the OpenRail Association will allow the library to be considered as a neutral solution and give more visibility to become the reference library for LRSs.
+
+## Are there competing products or projects? If there are please explain how the proposed projects differentiates.
+
+GIS software (like ARCGis and QGIS) have plugins to manipulate those data. However they cannot be used with existing software, as OSRD for example.
+
+## What standards does the project implement or rely on? How are they related to other existing standards?
+
+
+
+## What is the tech stack of the project? Name the major programming languages and frameworks which are used.
+
+The core is developped in rust.
```

Comment:
```suggestion
The core is developed in rust.
```
according to:
https://www.chicagomanualofstyle.org/search.html?clause=developed